openfisca_aotearoa/tests/social_security/disability_allowance/income_limits.yaml

```
@@ -0,0 +1,213 @@
+- name: Disability allowance - single person without dependant children, age b/w 16-17 years (Clause 10)
+  period: 2022-W07
+  absolute_error_margin: 0
+  input:
+    persons:
+      Lily:
+        date_of_birth: 2006-01-01
+        social_security__income: 
+          2022-W07: 200
+          2022-W08: 300
+          2022-W09: 589
+    families:
+      Whanau:
+        principal: Lily
+  output:
+    age:
+      2022-02-20: 16
+    social_security__in_a_relationship:
+      2022-W07: false
+      2022-W08: false
+      2022-W09: false
+    social_security__dependent_children:
+      2022-W07: 0
+      2022-W08: 0
+      2022-W09: 0
+    disability_allowance__income_limit_clause_10: 
+      2022-W07: true
+      2022-W08: true
+      2022-W09: false
+- name: Disability allowance - Any other single person, withount dependant children (Clause 11)
+  period: 2022-W07
+  absolute_error_margin: 0
+  input:
+    persons:
+      Lily:
+        date_of_birth: 2000-01-01
+        social_security__income: 
+          2022-W07: 589
+          2022-W08: 650
+          2022-W09: 735
+    families:
+      Whanau:
+        principal: Lily
+  output:
+    age:
+      2022-02-20: 22
+    social_security__in_a_relationship:
+      2022-W07: false
+      2022-W08: false
+      2022-W09: false
+    social_security__dependent_children: 
+      2022-W07: 0
+      2022-W08: 0
+      2022-W09: 0
+    disability_allowance__income_limit_clause_11: 
+      2022-W07: true
+      2022-W08: true
+      2022-W09: false
+- name: Disability allowance - Person in a relationship (Clause 12) without children
+  period: 2022-W07
+  absolute_error_margin: 0
+  input:
+    persons:
+      Lily:
+        date_of_birth: 2000-01-01
+        social_security__income: 
+          2022-W07: 200
+          2022-W08: 400
+          2022-W09: 600
+      Adam:
+        date_of_birth: 1998-01-01
+        social_security__income:
+          2022-W07: 200
+          2022-W08: 400
+          2022-W09: 600
+    families:
+      Whanau:
+        principal: Lily
+        partners: Adam
+  output:
+    age:
+      2022-02-20: [22, 24]
+    social_security__in_a_relationship: 
+    - true #Adam
```

I leave it to your own interpretation, but I propose nonetheless:

```suggestion
      [false, true]
```